### PR TITLE
Courses Wiki Link added

### PR DESCRIPTION
**Pull request by kartik1000**

### CODE REVIEW

**Comment by Ayushk4** [comment]
This looks okay to me. Just a couple of stuff.
- Name the commits well, not just the file which you are updating. You should also, mention the feature which you are adding or bugs you fixed.
- I think the location where you placed the text is a bit odd. I will let you know soon, where to shift/align the text.

**Comment by kartik1000** [comment]
After few changes I found that the Complete fill button was turning out to be more better than the outline one as a similar one has been used for the submit button. So I applied the complete fill one :)

**Comment by kartik1000** [comment]
Hey, In my commit above I made the appropriate changes so that we can directly reach the Course wiki page and not the search page and the Bootstrap button color fill is also there. Pls. review :)

**Comment by kartik1000** [comment]
Ok..I will correct them

**Comment by xypnox** [comment]
LGTM, I will merge after Testing it locally :+1:

**Comment by xypnox** [comment]
Perhaps we can store these names in a csv and search for them and then display the button only if the course page exists.

File: app.py [hunk 1]
```diff
@@ -28,6 +28,21 @@ def home():
...
+        #Extratcing name of course for wiki links
+        Course_info=code
+        #To extract Name of Course
+        Course_info=Course_info.split(":")[1][1:]
+        CourseName=Course_info
+        #Forming a string which can be used as a link
+        course_info_wiki_link=""
+        for i in range(0,len(Course_info)):
+            if(i==0):
+                course_info_wiki_link=course_info_wiki_link+"_"+Course_info[i]
```
Comment (Ayushk4) [inline app.py]: Must have a space before and after the assignment operator or any other operator.

File: app.py [hunk 2]
```diff
@@ -28,6 +28,21 @@ def home():
...
+            elif(Course_info[i]==" "):
+                course_info_wiki_link=course_info_wiki_link+"_"+Course_info[i+1]
```
Comment (Ayushk4) [inline app.py]: Sam as above.

**Comment by kartik1000** [comment]
Hey, I had to make the last two commits because the site was failing to load when we selected some courses which had no Coursename and had only CourseCode so now I have made changes so that it loads for them too

**Comment by kartik1000** [comment]
Yup..I realised that

**Comment by kartik1000** [comment]
Ok will work towards it👍

**Pull request closed without merging**